flink-runtime/src/main/java/org/apache/flink/runtime/io/network/partition/consumer/LocalInputChannel.java

```diff
@@ -260,6 +260,18 @@ void releaseAllResources() throws IOException {
 		}
 	}
 
+	int unSafeGetSizeOfQueuedBuffer() {
+		if (subpartitionView != null) {
+			try {
+				return subpartitionView.unsafeGetSizeOfQueuedBuffer();
+			} catch (Exception ignore) {
```

Comment:
What is this `catch` intended to solve? Potential  null pointer exception? If so it could be solved by:

```
			ResultSubpartitionView view = subpartitionView;
			if (view != null) {
				return view.unsynchronizedGetNumberOfQueuedBuffers();
			}
```